build/build.properties

```diff
@@ -18,3 +18,8 @@ project.javadoc                 = ${project.release}/javadoc
 project.zipdir                  = ${project.release}/${plugin.name}
 project.zipfilename             = ${plugin.name}.${plugin.version}.zip
 project.zipfilename.nightly     = ${plugin.name}.${plugin.version}-nightly.zip
+
+jameica.url                     = https://github.com/willuhn/jameica/archive/refs/tags
+jameica.version                 = V_2_10_4_BUILD_487
+hibiscus.url                    = https://github.com/willuhn/hibiscus/archive/refs/tags
+hibiscus.version                = V_2_10_24_BUILD_388
```

Comment:
Hier wird die neueste Version explizit angegeben. Mit renovate können wir automatisiert prs für neue Versionen erstellen lassen, sobald @willuhn einen neuen release tag erzeugt. Dank Github Actions werden die automatisch gebaut und wir könne sie bei Erfolg schnell integrieren.

Reply:
Könnte man die Tags auch in der `plugin.xml` bei `requires` mit speichern und hier als Variable nutzen? Dann wären die Dependencies an einer Stelle konfiguriert.

Reply:
Das wäre vermutlich möglich. Aber vermutlich nicht innerhalb des Schemas von Jameica. Die Tags haben einen anderen Aufbau als die Versionen der Plugins. Würde ich nicht machen wollen.

Reply:
> Könnte man die Tags auch in der `plugin.xml` bei `requires` mit speichern und hier als Variable nutzen? Dann wären die Dependencies an einer Stelle konfiguriert.

Ne, geht leider nicht, da es die [Schema Definition](https://www.willuhn.de/schema/jameica-plugin-1.5.xsd) nicht hergibt.